lib/internal/fs/promises.js

```diff
@@ -293,46 +290,43 @@ class FileHandle extends EventEmitter {
     if (options.type !== undefined) {
       validateString(options.type, 'options.type');
     }
+    if (options.type !== 'bytes') {
+      process.emitWarning(
+        'A non-"bytes" type option to readableWebStream() no longer does ' +
+        'anything. readableWebStream() always creates a byte-oriented stream ' +
+        'now.',
+        'ExperimentalWarning',
+      );
+    }
 
-    let readable;
+    const {
+      ReadableStream,
+    } = require('internal/webstreams/readablestream');
```

Comment:
Can you lazy load this?

Reply:
What do you mean? The require is already deferred until the last possible moment.

Reply:
Every time this function is called, it's re-required. Look at the lazy loads at the top of the file for examples of what I mean